flink-libraries/flink-table/src/main/scala/org/apache/flink/table/codegen/calls/FunctionGenerator.scala

```diff
@@ -414,6 +414,34 @@ object FunctionGenerator {
     DOUBLE_TYPE_INFO,
     BuiltInMethods.LOG_WITH_BASE)
 
+  addSqlFunctionMethod(
```

Comment:
Shift operations are core binary operators. I would not add them here but in `ScalarOperators`. Maybe we can already reuse `generateArithmeticOperator`.

Reply:
Yes, that's correct..
+1 to add `shift` as a binary operator.